project/LauncherConfigs.scala

```diff
@@ -61,28 +57,6 @@ object LauncherConfigs {
       }
     )
 
-  /**
-   * react-native prod bundler needs require function without name spaces
```

Comment:
we need those process functions. react-native asset system uses `require`

example : 
```scala
    @inline def load[T](lib: String): T = g.require(lib).asInstanceOf[T]

    @inline def loadDynamic(lib: String): js.Dynamic = load[js.Dynamic](lib)

loadDynamic("./images/thumbnails/dislike.png"),

```

in production mode we need to remove prefix etc